service/prometheus/group_test.go

```diff
@@ -15,7 +15,7 @@ func Test_Prometheus_FilterInvalidServices(t *testing.T) {
 		services         []v1.Service
 		expectedServices []v1.Service
 	}{
-		// Test a service without either cluster or certificate annotations is filtered.
+		// Test a service without a cluster annotation is filtered.
```

Comment:
There is a fair amount of test movement due to removing the certificate annotation. The diff sucks.